src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppElasticBeanstalk.recipe

```diff
@@ -139,11 +140,67 @@
             "Id": "InstanceType",
             "Name": "EC2 Instance Type",
             "Description": "The EC2 instance type of the EC2 instances created for the environment.",
-            "Type": "String",
+            "Type": "Object",
             "TypeHint": "InstanceType",
-            "DefaultValue": "",
             "AdvancedSetting": true,
-            "Updatable": true
+            "Updatable": true,
+            "ChildOptionSettings": [
+                {
+                    "Id": "InstanceFreeTierEligible",
```

Comment:
I don't think this is right to add these settings like InstanceFreeTierEligible, InstanceArchitecture, InstanceCPUCores and InstanceMemory that are not actual settings by filter for the typehint.

Reply:
Done